tests/pages/updateIdentity.test.tsx

```diff
@@ -1,27 +1,28 @@
-jest.mock("../../src/hooks/useCookies");
+jest.mock("@src/hooks/useCookies");
 
 import { mount } from "enzyme";
 import fetch from "jest-fetch-mock";
 import { enableFetchMocks } from "jest-fetch-mock";
 import React from "react";
 import { ThemeProvider } from "styled-components";
 
-import { ReceivedIdentityTypes, Identity } from "../../src/@types/Identity";
-import { Layout } from "../../src/components/Layout";
-import { Input } from "../../src/components/display/fewlines/Input/Input";
+import { ReceivedIdentityTypes, Identity } from "@src/@types/Identity";
+import { Layout } from "@src/components/Layout";
+import { Input } from "@src/components/display/fewlines/Input/Input";
+import { NavigationBreadcrumbs } from "@src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs";
 import {
   UpdateIdentityForm,
   Form,
-} from "../../src/components/display/fewlines/UpdateIdentityForm";
-import { GlobalStyle } from "../../src/design-system/globals/globalStyle";
-import { lightTheme } from "../../src/design-system/theme/lightTheme";
-import { useCookies } from "../../src/hooks/useCookies";
-import UpdateIdentityPage from "../../src/pages/account/logins/[type]/[id]/update";
-import * as fetchJson from "../../src/utils/fetchJson";
+} from "@src/components/display/fewlines/UpdateIdentityForm";
+import { GlobalStyle } from "@src/design-system/globals/globalStyle";
+import { lightTheme } from "@src/design-system/theme/lightTheme";
+import { useCookies } from "@src/hooks/useCookies";
+import UpdateIdentityPage from "@src/pages/account/logins/[type]/[id]/update";
+import * as fetchJson from "@src/utils/fetchJson";
 
 enableFetchMocks();
 
-jest.mock("../../src/config", () => {
+jest.mock("@src/config", () => {
```

Comment:
Hmm, I was wondering if this was working 😅.

Reply:
It does 😜